pkg/mkimage-raw-efi/install

```diff
@@ -168,12 +168,20 @@ zfs_umount() {
   umount /root/dev ||:
 }
 
+
+cmdline="/proc/cmdline"
 logmsg "EVE-OS installation started"
+if grep -q interactive /proc/cmdline; then
+  logmsg "Interactive mode"
+  bash ./parse-interactive-installer-conf
```

Comment:
Is there a case when the interactive installer onboot container ran, but did not produce a config file? Should you check for its output here?

Reply:
There is no such case, at the very least it will have the contents of /proc/cmdline.